src/components/weather/styled.ts

```diff
@@ -28,3 +57,114 @@ export const LoadingText = styled.p`
   margin-top: 20px;
   font-style: italic;
 `;
+
+export const SavedCities = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+export const SavedCity = styled.div`
+  background-color: white;
+  border-radius: 5px;
+  padding: 10px;
+  margin: 5px;
+  width: 150px;
+  text-align: center;
+`;
+
+export const RemoveButton = styled.button`
+  background-color: #ff5050;
+  color: white;
+  border: none;
+  padding: 5px 10px;
+  border-radius: 5px;
+  cursor: pointer;
+`;
+
+export const SaveButton = styled.button`
+  background-color: #66cc00;
+  color: white;
+  border: none;
+  padding: 5px 10px;
+  border-radius: 5px;
+  cursor: pointer;
+`;
+
+export const CurrentWeatherWrapper = styled.div`
+  background-color: white;
+  border-radius: 10px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+  text-align: center;
+  margin: auto;
+  width: 300px; 
+  height: 350px; 
+  max-width: 800px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const CurrentWeatherStatus = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  width: 25rem;
+
+  @media (min-width: 600px) {
+    flex-direction: row;
+    align-items: center;
+  }
+`;
+
+export const SyledCityTitle = styled.h4`
+font-family: 'Verdana', sans-serif;
+  font-size: 24px;
+   font-weight: 700;
+    font-size: 1.25rem;
+    color: #396bae;
+`;
+
+export const WeatherIcon = styled.div`
+  margin: 16px 0;
+  
+  & > p > img {
+    max-width: 100%;
+    height: auto;
+    width: 100px; 
+    height: 70px; 
+  }
+`;
+
+export const InfoWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-left: 16px; 
+
+  @media (min-width: 600px) {
```

Comment:
600 px is not standard breakpoint, you only use for some specific cases.

Please use like this

```

  mobile: 0 767,
  tablet: 768 1023,
  desktop: 1024 - 
```